filer/templates/admin/filer/folder/file_row.html

```diff
@@ -2,7 +2,7 @@
 <td class="center">
 
     {% if is_popup and not select_folder %}
-        <a class="insertlink insertlinkButton" href="" onclick="opener.dismissRelatedImageLookupPopup(window, {{ file.id }}, '{{ file.icons.32|escapejs }}', '{{ file.label|escapejs }}'); return false;" title="{% trans "Select this file" %}"><i class="ace-icon fa fa-long-arrow-up bigger-125 blue"></i></a>
+        <a class="insertlink insertlinkButton" href="" onclick="opener.dismissRelatedImageLookupPopup(window, {{ file.id }}, '{{ file.icons.32|escapejs }}', '{{ file.label|escapejs }}', '{{ file.url }}'); return false;" title="{% trans "Select this file" %}"><i class="ace-icon fa fa-long-arrow-up bigger-125 blue"></i></a>
```

Comment:
`file.url` is unlikely to have quotes in it, but it wouldn't hurt to use `|escapejs` as a precaution.